src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -555,7 +555,9 @@ private async Task<PackageSpec> GetPackageSpecAsync(ISettings settings)
                     CentralPackageFloatingVersionsEnabled = MSBuildStringUtility.IsTrue(_vsProjectAdapter.BuildProperties.GetPropertyValue(ProjectBuildProperties.CentralPackageFloatingVersionsEnabled)),
                     CentralPackageTransitivePinningEnabled = MSBuildStringUtility.IsTrue(centralPackageTransitivePinningEnabled),
                     RestoreAuditProperties = auditProperties,
-                }
+                },
+                SdkAnalysisLevel = _vsProjectAdapter.BuildProperties.GetPropertyValue(ProjectBuildProperties.SdkAnalysisLevel),
+                UsingMicrosoftNETSdk = IsSdk,
```

Comment:
So, a few weeks/months ago, we had a customer who is implementing their own custom project system in VS, use an SDK style project but NuGet used the project via `LegacyPackageReferenceProejct` because they weren't using CPS. Therefore, I think it's safer to get this property via the project's build properties service, rather than assuming it's false.